policies/expires_after_tagging/README.md

```diff
@@ -0,0 +1,9 @@
+# Expires After Tagging Policy
```

Comment:
can you name the policy with `-` in between words instead of `_`, more of a normal folder naming convention